ceph.spec.in

```diff
@@ -47,14 +47,14 @@
 %endif
 %endif
 %bcond_with seastar
-%if 0%{?suse_version} >= 1500
+%if 0%{?fedora} || 0%{?suse_version} >= 1500 # distros where py2 is _optional_
```

Comment:
```
error: /tmp/install-deps.1467/ceph.spec:50: bad %if condition
```

Reply:
probably we should put comment in separated lines?

Reply:
gah, yes